apps/qa/requirements.txt

```diff
@@ -1,4 +1,3 @@
-./dcpy
```

Comment:
using pyproject.toml instead to not install tests. Figured we should be consistent about how it gets installed in different environments